packages/api-providers/src/abstracts/relayer/types.ts

```diff
@@ -70,6 +72,44 @@ export type RelayerConfig = {
   endpoint: string;
 };
 
+export type MixerQuery = {
+  amount: number;
+  tokenSymbol: string;
+};
+
+export type RelayedChainInput = {
```

Comment:
Docs on what this is and what it is used for are helpful.

Reply:
Added a comment